Email Notifications

gigabite-pro opened the pull request:
- Added email notifications for various events listed in the google sheet.
- Please ask for the google sheet on discord.
- Green highlight means that it has been implemented. 
- Yellow ones have also been implemented but not on the click of the button (since no one created the button "Notify Instructor").
- Remaining ones may not be top priority (need confirmation on this).

jjessieshang commented:
I see your commit for calendar bug fix, but the weekly view still isn't rendering - were these changes overwritten?

gigabite-pro commented:
I think the calendar code got overwritten but I fixed that. Also, I removed react-weekview so now the calendar updates if you select a random date from the date-selector. iImade my own hook to manage the week data

jjessieshang commented:
Looks good to me! Calendar update on date select is nice. minor detail lets have the "Today" text beside the Month text match in font and font weight
